config/locales/ja.yml

```
@@ -0,0 +1,188 @@
+        try: 次やること
+  date:
+    abbr_day_names:
+    - 日
+    - 月
+    - 火
+    - 水
+    - 木
+    - 金
+    - 土
+    abbr_month_names:
+    - 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)